Docker/Dockerfile

```diff
@@ -0,0 +1,136 @@
+FROM adoptopenjdk:8u282-b08-jdk-openj9-0.24.0-focal
+
+RUN apt-get update
+RUN apt-get upgrade -y
+
+########## cmocka ##########
+RUN apt-get install -y libcmocka-dev
+
+########## ikos Dockerfile ##########
+MAINTAINER Maxime Arthaud <maxime.arthaud@nasa.gov>
+ARG njobs=2
+ARG build_type=Release
+
+
+# Install all dependencies
+RUN apt-get install -y gnupg gnupg2
+RUN apt-get install -y gcc g++ cmake libgmp-dev libboost-dev
+RUN apt-get install -y libboost-filesystem-dev libboost-thread-dev libboost-test-dev python
+RUN apt-get install -y python-pygments libsqlite3-dev libtbb-dev libz-dev libedit-dev
+RUN apt-get install -y llvm-9 llvm-9-dev llvm-9-tools clang-9 wget
+
+# Add ppa for llvm 9.0
+RUN echo "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-9 main" >> /etc/apt/sources.list
+
+# Add llvm repository key
+RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
+
+# Refresh cache
+RUN apt-get update
+
+########## Jenkins Dockerfile ##########
+
+RUN apt-get upgrade -y \
+  && apt-get install -y --no-install-recommends git curl fontconfig unzip vim sudo clang-format-10\
+  && apt install -y maven \
+  && curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash && apt-get install -y git-lfs && git lfs install \
```

Comment:
Please remove the backslash character at the end of L36. It leads to "Empty continuation lines".